src/frontend/js/pages/TeacherDashboardContractsLayout/hooks/useDownloadContractArchive/contractArchiveLocalStorage.ts

```diff
@@ -0,0 +1,51 @@
+export const CONTRACT_ARCHIVE_LOCAL_STORAGE_KEY = 'contractArchiveId';
```

Comment:
By convention, all our local storage application keys are prefixed by `RICHIE_` and uppercased.

```suggestion
export const CONTRACT_ARCHIVE_LOCAL_STORAGE_KEY = 'RICHIE_CONTRACT_ARCHIVE';
```